src/SpectatorView.Unity/Assets/SpectatorView/Scripts/StateSynchronization/StateSynchronizationObserver.cs

```diff
@@ -20,9 +21,13 @@ public class StateSynchronizationObserver : NetworkManager<StateSynchronizationO
         public const string AssetBundleRequestInfoCommand = "RequestAssetBundleInfo";
         public const string AssetBundleReportInfoCommand = "ReportAssetBundleInfo";
         public const string AssetBundleRequestDownloadCommand = "RequestAssetBundleDownload";
-        public const string AssetBundleReportDownloadCommand = "ReportAssetBundleDownload";
+        public const string AssetBundleReportDownloadStartCommand = "ReportAssetBundleDownloadStart";
+        public const string AssetBundleReportDownloadDataCommand = "ReportAssetBundleDownloadData";
         public const string AssetLoadCompletedCommand = "AssetLoadCompleted";
 
+        public const string AssetBundleName = "spectatorview";

```

Comment:
>spectatorview [](start = 47, length = 13)

it seems like we should include some sort of indication that this is an asset bundle. maybe "SpectatorView.AssetBundle" or something?